homeassistant/components/binary_sensor/xiaomi_aqara.py

```diff
@@ -119,7 +125,7 @@ def parse_data(self, data, raw_data):
         if value is None:
             return False
 
-        if value == '1':
+        if value in ['1', '2']:
```

Comment:
I introduced an additional value (2: Manual alarm) here, The same applies to the Smoke/Gas sensor.

Reply:
```python
if value in '1', '2':
```